askama_shared/src/generator.rs

```diff
@@ -1230,13 +1230,19 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
     fn visit_lit(&mut self, lws: &'a str, val: &'a str, rws: &'a str) {
         assert!(self.next_ws.is_none());
         if !lws.is_empty() {
-            if self.skip_ws {
-                self.skip_ws = false;
+            if self.skip_ws == WhitespaceHandling::Suppress {
```

Comment:
Can we do a `match` here somehow? I don't remember exactly how `self.skip_ws` and `val` interact.

Reply:
I replaced this with a `match`. Please tell me if it looks good to you.